lib/routes/home/widgets/bottom_actions_bar/enter_payment_info_dialog.dart

```diff
@@ -208,13 +208,15 @@ class EnterPaymentInfoDialogState extends State<EnterPaymentInfoDialog> {
     final texts = context.texts();
     try {
       _setValidatorErrorMessage("");
-      // TODO: Liquid - Previously parseInput, parseInvoice only parses LNInvoice's.
-      final inputType = context.read<InputBloc>().parseInvoice(input: input);
+      final inputType = context.read<InputBloc>().parseInput(input: input);
       _log.info("Parsed input type: '${inputType.runtimeType.toString()}");
       // Can't compare against a list of InputType as runtime type comparison is a bit tricky with binding generated enums
-      // TODO: Liquid - Add other supported InputType's once parse_invoice has evolved into parse_input.
-      // ignore: unnecessary_type_check
-      if (inputType is! LNInvoice) {
+      if (!(inputType is InputType_Bolt11 ||
+          inputType is InputType_LnUrlPay ||
+          inputType is InputType_LnUrlWithdraw ||
+          inputType is InputType_LnUrlAuth ||
+          inputType is InputType_LnUrlError ||
+          inputType is InputType_NodeId)) {
```

Comment:
I think NodeId would also not be supported

Reply:
Noted. It'll be removed as part of #30